app/models/user.rb

```diff
@@ -1,8 +1,35 @@
+require "net/http"
+require "uri"
+
 class User < ActiveRecord::Base
   has_secure_password
 
+  enum authy_status: [:unverified, :onetouch, :sms, :token, :approved, :denied]
   validates :email,  presence: true, format: { with: /\A.+@.+$\Z/ }, uniqueness: true
   validates :name, presence: true
   validates :country_code, presence: true
   validates :phone_number, presence: true
+
+  def send_one_touch
```

Comment:
I don't feel as though this method should be part of the User model. Would have thought that the controller was the place for this. Makes things more explicit with regards to updating the user and where it happens.